specification/configuration.md

```diff
@@ -222,7 +222,7 @@ are required.
 - Zipkin exporter
   - Distribution MUST NOT list Zipkin exporter as supported (not supported by Smart Agent)
 - `OTEL_TRACES_EXPORTER`
-  - Non-RUM distribution MUST default to `otlp` over gRPC with an endpoint of `localhost:4317`
+  - Non-RUM distribution MUST default to `otlp` using `grpc` or `http/protobuf` transport protocol.
```

Comment:
This loses the default endpoint specification. In other words, it used to be clear from reading this what wire format and protocol should be used, but also where it should be sent. Now it just says what format and protocols might be used, but not where to send it.

Can we get default endpoints specified?

Reply:
> Remove the default value as this is the same on as [specified in Otel](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md)

Reply:
Makes sense, thanks for clarifying. It's redundant to add it here. 👍🏻